zgw/zaken-api/src/main/kotlin/nl/nlportal/zakenapi/client/request/ZaakRollenImpl.kt

```diff
@@ -51,13 +51,8 @@ class SearchRollenImpl(val zakenApiClient: ZakenApiClient) : SearchZaakRollen {
         kvkNummer: String,
         vestigingsNummer: String,
     ): SearchZaakRollen {
-        queryParams.add("rol__betrokkeneIdentificatie__vestiging__vestigingsNummer", vestigingsNummer)
-        queryParams.add("rol__betrokkeneIdentificatie__vestiging__kvkNummer", kvkNummer)
-        return this
-    }
-
-    override fun ofVestigingsNummer(vestigingsNummer: String): SearchZaakRollen {
```

Comment:
Let's not remove functionality in a hotfix that has just been released, even if it is unused.

Please only update the bodies of these two functions to apply their respective queryParams and keep both functions.